docs/firefox-accounts.rst

```diff
@@ -37,6 +37,18 @@ predictability in retroactive analysis.
     <https://mozilla.github.io/application-services/docs/accounts/metrics.html#descriptions-of-metrics-related-query-parameters>`_
     rules applied by the Firefox Accounts server.
 
+UITour Flow
+-----------
+
+Since Firefox 79 the FxA links and email forms use the UITour to show the Firefox Accounts page and log the browser
+into Sync or an Account. For non-Firefox browsers or if the UITour is not available the flow uses normal links that
```

Comment:
> For non-Firefox browsers or if the UITour is not available the flow uses normal links that
allow users to log into FxA as a website only without connecting the Firefox Desktop client.

I'm curious as to what the behaviour should be here, as right now if UITour isn't available we still have all the legacy logic in place to support signing in to Sync for older Firefox versions. I'm assuming we want to keep this, so we should probably make that clearer here?

Reply:
> if UITour isn't available we still have all the legacy logic in place to support signing in to Sync for older Firefox versions.

I don't know what the deprecation strategy is in place right now. My thoughts on this were: we can support the legacy logic for 1 or 2 ESRs + keep an eye on FxA metrics and see where the logins are coming from. Once it is down to almost zero we can remove the legacy flows. I will update the docs above